main/ui/src/main/resources/fxml/welcome.fxml

```diff
@@ -24,6 +24,14 @@
 		<HBox alignment="CENTER" spacing="5.0" cacheShape="true" cache="true">
 			<Label fx:id="checkForUpdatesStatus" cacheShape="true" cache="true" />
 			<ProgressIndicator fx:id="checkForUpdatesIndicator" progress="-1" prefWidth="15.0" prefHeight="15.0" cacheShape="true" cache="true" cacheHint="SPEED" />
+			<Label fx:id="checkForUpdatesErrorLabel" />
+			<Button fx:id="checkForUpdatesRetryButton" cache="true" cacheShape="true" defaultButton="true" onAction="#executeCheckForUpdates" text="%welcome.checkForUpdates.retry">
+				<graphic>
```

Comment:
Don't use a png for this, we have imported the [ionicons](http://ionicons.com/) font for these purposes:

```
<graphic><Label text="&#xf21c;" styleClass="ionicons"/></graphic>
```